azure-pipelines.yml

```diff
@@ -110,8 +112,20 @@ jobs:
         displayName: yarn
 
       - script: |
-          NODE_ENV=production yarn test:fluentui:visual
-        displayName: run FUI VR Test
+          NODE_ENV=production yarn test:fluentui:visual:build
+        displayName: build FUI N* VR Test
+
+      - task: AzureUpload@1
+        displayName: Upload PR deploy site
+        inputs:
+          SourcePath: 'packages/fluentui/docs/dist'
+          azureSubscription: 'UI Fabric (bac044cf-49e1-4843-8dda-1ce9662606c8)'
+          storage: fabricweb
+          ContainerName: '$web'
+          BlobPrefix: 'pr-deploy-site/$(Build.SourceVersion)/react-northstar'
```

Comment:
Will this conflict with the site being uploaded in the main version of PR deploy? Also can you use the variable here?

Reply:
With added `.devops/templates/variables.yml` it's more visible. With current implementation it will not collide as for PR deploy we use PR's ref while this task uses commits' hashes. _And it may be overkill..._

---

1. There is no way to cancel jobs in Screener via API, it's visible even in `screener-runner` - it waits until the previous job will be finished
2. Screener uses commit SHA to build a connection between builds

If we will decide to go with PR refs instead we can receive weird race conditions when we build and upload a new artifact before the previous job will end.

Please let me know if you have ideas/suggestions.